packages/react-devtools-shared/src/__tests__/inspectedElementSerializer.js

```diff
@@ -1,12 +1,20 @@
 // test() is part of Jest's serializer API
 export function test(maybeInspectedElement) {
+  if (
+    maybeInspectedElement === null ||
+    typeof maybeInspectedElement !== 'object'
+  ) {
+    return false;
+  }
+
+  const hasOwnProperty = Object.prototype.hasOwnProperty.bind(
+    maybeInspectedElement,
+  );
```

Comment:
This too.

Reply:
Same approach as https://github.com/facebook/react/pull/20618#discussion_r609958871. Does this resolve this for you?